biz.aQute.repository/src/aQute/maven/api/Archive.java

```diff
@@ -17,6 +17,10 @@
 			+ "\\s*",																																																			// skip
 																																																								// whitespace
 		Pattern.COMMENTS);
+
+	public static final String	SOURCES_CLASSIFIER	= "sources";
+	public static final String	JAVADOC_CLASSIFIER	= "javadoc";
```

Comment:
This is a good idea. Perhaps also add `JAR_EXTENSION = "jar"` to use instead of `"jar"` everywhere?

Reply:
because you ask so nicely ... :-)